lib/segment/src/index/hnsw_index/graph_links.rs

```diff
@@ -103,6 +132,7 @@ impl GraphLinksFileHeader {
             levels_count: arr[1],
             total_links_len: arr[2],
             total_offsets_len: arr[3],
+            offsets_padding: arr[4],
```

Comment:
Feels like this might need some kind of migration?

Or do we have a fixed number of bytes (which is more than `5 * size_of::<u64>()` in this case) allocated for the header, and all unused bytes are set to 0, and so if we try to load an older file, with new code, `arr[4]` will be 0, and all will be good? 🤔

Reply:
Answered if PR description. There is a reserved data for future header extension

Reply:
![image](https://github.com/qdrant/qdrant/assets/1935623/32d05b54-5065-4625-9658-36fca0b146fd)